internal/quickstart/toggle_flag.go

```diff
@@ -26,6 +32,14 @@ type toggleFlagModel struct {
 	helpKeys       keyMap
 	sdkKind        string
 	spinner        spinner.Model
+
+	// Throttling fields to control how quickly a user can press (or hold) tab to toggle the flag.
```

Comment:
Hopefully this is helpful to explain what's going on since there are two throttling mechanisms. When I just used `throttleCount` that's incremented and passed around in a message, it would eventually "catch" up and start updating sooner than I wanted. Using `resetThrottleCount` that's reset whenever we stop throttling helps prevent unexpected updates going through.